tests/common_testing/alfasim_sdk_common_testing/filled_case_descriptions.py

```diff
@@ -368,12 +368,35 @@
     flow_rates=Array([1.0] * 4 + [0.05] * 4, "m3/s"),
     pressure_boosts=Array([1.0, 2.0, 3.0, 4.0, 5.0, 6.0, 7.0, 8.0], "bar"),
 )
-TREND_OUTPUT_DESCRIPTION = case_description.TrendOutputDescription(
+POSITIONAL_TREND_OUTPUT_DESCRIPTION = case_description.PositionalTrendDescription(
     curve_names=["oil mass flow rate"],
     element_name="pipe 1",
     position=Scalar(2950.0, "m"),
     location=constants.OutputAttachmentLocation.Main,
 )
+GLOBAL_TREND_OUTPUT_DESCRIPTION = case_description.GlobalTrendDescription(
+    curve_names=["oil mass flow rate"],
```

Comment:
This is actually an invalid curve for global trends. Use `timestep`

Reply:
Ops, nice catch! Although I think it don't check for the curve names in `alfasim-sdk` tests.

Reply:
Yep, I think the conversion is only checked on alfasim tests. But since these can be used as examples, it is nice to have them with valid properties.